index.html

```diff
@@ -22,20 +22,43 @@
   <body>
     <div class="after-bdy">
       <div class="portfolio-class">
-        <nav class="flex white-back">
+        <div id="mobile-menu" class="mobile-menu">
+          <div class="menu-cross">
+            <img
+              onclick="myFunction()"
+              class="cross-btn"
+              src="images/menu-cross.png"
+              alt=""
+            />
+          </div>
+
+          <ul class="mobile-menu-items">
+            <li>
+              <a onclick="myFunction()" href="#portfolio-id">Portoflio</a>
+            </li>
+            <li><a onclick="myFunction()" href="#about-id">About</a></li>
+            <li><a onclick="myFunction()" href="#contact-me-id">Contact</a></li>
```

Comment:
- [ ] When I click `about` it sends me here which not the correct part of the page, kindly update it 
![image](https://user-images.githubusercontent.com/62705947/195055350-1dbcc3e8-6d9e-4de4-9cd5-2b3ff6b6fe47.png)